examples/purchaseTesterTypescript/app/APIKeys.tsx

```diff
@@ -1,9 +1,8 @@
-
 // REPLACE THESE WITH YOUR OWN
 const APIKeys = {
-  apple: "",
-  google: "",
-  amazon: ""
+  apple: 'appl_KFpRDwauDyfIGIETKyVymslMcjX',
+  google: 'goog_cRtaFpfwfcQYPXkIEqFUkVszUfN',
```

Comment:
Need to remove :)

Reply:
We need a git commit hook to prevent this 🙈